tutorials/Bayes/TA_solutions/BayesDay_Tutorial_1_solutions.ipynb

```diff
@@ -3,7 +3,7 @@
   "nbformat_minor": 0,
```

Comment:
<p>this is hot garbage</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NeuromatchAcademy/course-content/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>